lib/stdlib/doc/src/orddict.xml

```diff
@@ -101,6 +140,17 @@
           the <c><anno>Key</anno></c> is present in the dictionary. An exception
           is generated if <c><anno>Key</anno></c> is not in the dictionary.</p>
         <p>See also section <seeerl marker="#notes">Notes</seeerl>.</p>
+        <p><em>Example 1:</em></p>
+<pre>
+1> <input>OrdDict1 = orddict:from_list([{a, 1}, {b, 2}]).</input>
+2> <input>orddict:fetch(a, OrdDict1).</input>
```

Comment:
For consistency with other examples, you should show the output of the command. (That applies also to other some other examples.)

Reply:
Yes, I agree. Will do tomorrow